### PR TITLE
Fix: Use local instance of shelljs

### LINKED ISSUE #1781: Use local instance of shelljs vs shelljs/global in scripts/clean.js
[scripts/clean.js](https://github.com/react-boilerplate/react-boilerplate/blob/master/internals/scripts/clean.js) uses `shelljs/global` which is a [non-recommended](https://github.com/shelljs/shelljs#global-vs-local) way. Apparently it pollutes the global namespace.

- Was there any specific reason to use the global shelljs?
- If not, can we switch to local instance of shelljs?

### PR DESCRIPTION
**Pull request by KarandikarMihir**
Nothing crazy. Just fixes #1781

### CODE REVIEW

File: internals/scripts/clean.js [hunk 1]
```diff
@@ -1,58 +1,58 @@
-require('shelljs/global');
+var shell = require('shelljs');
```
Comment (Dattaya) [inline internals/scripts/clean.js]: nit: you can use `const` instead of `var` the same way as `addCheckMark` declared

**Comment by Dattaya** [comment]
I don't like that global require too. Usually in most of programming languages global imports are discouraged. 👍 from me.

**Comment by KarandikarMihir** [comment]
@Dattaya I feel stupid! :sweat_smile: Should be good now.

**Comment by Dattaya** [comment]
💯 Thanks, going to merge as soon as AppVeyor completes the build. 😃

**Comment by Dattaya** [comment]
Appveyor's stuck and somehow I don't have admin access, going to close and reopen to bring it to life

**Pull request merged**